packages/cumulus-api-client/tests/test-CumulusApiClient.js

```diff
@@ -38,18 +48,30 @@ test.before(async (t) => {
 });
 
 test.after.always(
-  () => dynamodb().deleteTable({ TableName: process.env.tableName }).promise()
+  async () => dynamodb().deleteTable({ TableName: process.env.tableName }).promise()
 );
 
+test.serial('getCacheAuthToken initializes cache and sets token if cacheInitialized is false', async (t) => {
+  const token = randomId();
+  const testApiClient = new CumulusApiClientRewire(t.context.config);
+  testApiClient.createNewAuthToken = async () => token;
+  testApiClient._validateTokenExpiry = async () => true;
+  const actual = await testApiClient.getCacheAuthToken();
+  t.is(token, actual);
+  t.is(true, testApiClient.cacheInitialized);
```

Comment:
Nit: you can do `t.true(testApiClient.cacheInitialized)`

Reply:
df015f064